libr/include/r_skiplist.h

```diff
@@ -0,0 +1,34 @@
+// (c) 2016 Jeffrey Crowell
+// BSD 3 Clause License
+// radare2
+
+// Skiplists are a probabilistic datastructure than can be used as a k-v store
+// with average case O(lg n) lookup time, and worst case O(n).
+
+// https://en.wikipedia.org/wiki/Skip_list
+
+#ifndef R2_SKIP_LIST_H
+#define R2_SKIP_LIST_H
+
+#include <r_list.h>
+
+typedef struct r_skiplist_node_t {
+	void *data;	// pointer to the value
+	struct r_skiplist_node_t *forward[1]; // forward pointer
```

Comment:
why forward[1] shouldn't it be kSkipListDepth or at least `struct r_skiplist_node_t **forward`?